applications/dashboard/models/class.usermodel.php

```diff
@@ -1838,7 +1839,10 @@ public function getRoles($userID) {
 
         $result = [];
         foreach ($rolesDataArray as $roleID) {
-            $result[] = RoleModel::roles($roleID, true);
+            $role = RoleModel::roles($roleID, $includeInvalid);
```

Comment:
I went to check on that second parameter for `RoleModel::roles()` and it just says `$force`. Would you mind filling out the docblock of `RoleModel::roles()` as part of this PR?

Reply:
Done. I targeted the wrong branch (master) with the original docs commit, but they're here now.